resources/test/fixtures/flake8_import_conventions/from_imports.py

```diff
@@ -0,0 +1,11 @@
+# Test absolute imports
+import xml.dom.minidom as wrong
+from xml.dom import minidom as wrong
+from xml.dom.minidom import parseString as wrong  # Ensure ICN001 throws on function import.
+
+import xml.dom.minidom as md
+from xml.dom import minidom as md
+from xml.dom.minidom import parseString as pstr
+
+# Test relative imports
+from ..xml.dom import minidom as okay  # Ensure config "xml.dom.minidom" doesn't catch relative imports
```

Comment:
Per your comment in the issue -- yeah, not sure what should happen if a user has `"..xml.dom.minidom"` in their `pyproject.toml`. It's obviously a bit weird to support relative imports there ("Relative to what?"), but I think it's fine as-is. There's not a clear right answer. The alternative would be to avoid calling `check_conventional_import` for relative imports at all.

Reply:
Leaving as is.